functions/src/api/v1/users/getUser.ts

```diff
@@ -0,0 +1,57 @@
+import { Request, Response } from 'express'
+import { ForbiddenError } from '../../errors/HttpError'
+import { getUserDb } from '../../../db/getUserDb'
+
+/**
+ * @swagger
+ * /v1/users/{userId}:
+ *   get:
+ *     description: Get user details, subscription and quota status. This will not send any payment details or API keys.
+ *     tags: ['users']
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: User details
+ *         content:
+ *           application/json:
+ *             example: {
+ *               id: 'vXIUuaGkH4kukbwRH5cU',
+ *               updatedAt: 1605969562,
+ *               subscription: {
+ *                   isActive: true,
+ *                   isQuotaReached: false
+ *               },
+ *               sentEmails: 1,
+ *               sentSMSs: 2
+ *             }
+ *             schema:
+ *               type: object
+ *               $ref: '#/components/schemas/User'
+ *       401:
+ *         description: missing authorization bearer token
+ *       403:
+ *         description: authorization header present but not valid
+ *       412:
+ *         description: authorization header present but not formatted correctly
```

Comment:
pls consider simplifying the two above with:
- `invalid authorization header`
- `authorization header wrong format`